Blueprints/BlueprintDefinitions/vs2022/ChatBotTutorial/template/src/BlueprintBaseName.1/FlowerOrder.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace BlueprintBaseName._1
+{
```

Comment:
nit: may be we can use less new lines.

Reply:
Once we move have .NET 6 support in all of the templates I want to revamp them to use newer styles like filescope namespaces. For now I don't want to touch them.